### PR TITLE
feat: add view employee component

### PR DESCRIPTION
**Pull request by Rishabh106**
## Description
Added routing to the rows in Employee data table, implemented the API and designed the UI.

## Related issues and discussion
#534 

## Screenshots, if any
view_employee
view_employee2

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by Rishabh106** [comment]
For improvement in presentation, I changed the Active - true/false to Status - Active/Inactive and isLoanOfficer - true/false to Yes/No. Note: this is a deviation from the community app.

File: src/app/organization/organization.service.ts [hunk 1]
```diff
@@ -55,6 +55,14 @@ export class OrganizationService {
     return this.http.post('/staff', employee);
   }
 
+  /**
+   * @param {string} staffId Staff ID of employee.
```
Comment (abhaychawla) [inline src/app/organization/organization.service.ts]:
```suggestion
   * @param {string} employeeId Employee ID
```

Update the parameter.

File: src/app/organization/employees/view-employee/view-employee.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { Observable } from 'rxjs';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Employee data resolver.
+ */
+@Injectable()
+export class ViewEmployeeResolver implements Resolve<Object> {
```
Comment (abhaychawla) [inline src/app/organization/employees/view-employee/view-employee.resolver.ts]:
```suggestion
export class EmployeeResolver implements Resolve<Object> {
```

Rename the file as well and update all the references as well.

File: src/app/organization/employees/view-employee/view-employee.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+   * @param {OrganizationService} organizationService Organization service.
+   */
+  constructor(private organizationService: OrganizationService) {}
+
+  /**
+   * Returns the employee data.
+   * @returns {Observable<any>}
+   */
+  resolve(route: ActivatedRouteSnapshot): Observable<any> {
+    const id = route.paramMap.get('id');
```
Comment (abhaychawla) [inline src/app/organization/employees/view-employee/view-employee.resolver.ts]:
```suggestion
    const id = route.paramMap.get('id');
```
```suggestion
    const employeeId = route.paramMap.get('id');
```

File: src/app/organization/employees/view-employee/view-employee.component.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+        <div fxFlex="50%" *ngIf="employeeData.mobileNo">
+          {{ employeeData.mobileNo }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Status
+        </div>
+
+        <div fxFlex="50%" [className]="employeeData.isActive === true ? 'true' : 'false'">
```
Comment (abhaychawla) [inline src/app/organization/employees/view-employee/view-employee.component.html]:
```suggestion
        <div fxFlex="50%">
```
No class is being applied on the basis of this condition.

File: src/app/organization/employees/view-employee/view-employee.component.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+        <div fxFlex="50%">
+          {{ employeeData.officeName }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Is Loan Officer
+        </div>
+
+        <div fxFlex="50%" [className]="employeeData.isLoanOfficer === true ? 'true' : 'false'">
```
Comment (abhaychawla) [inline src/app/organization/employees/view-employee/view-employee.component.html]:
```suggestion
        <div fxFlex="50%">
```
No class is being applied on the basis of this condition.

File: src/app/organization/employees/view-employee/view-employee.component.html (hunk 4 above)
Comment (Rishabh106) [reply src/app/organization/employees/view-employee/view-employee.component.html]: My Bad, I thought a class name was required to use the conditional operator but turns out, it does not need one.

**Pull request merged**